runtime/util/vmargs.c

```diff
@@ -506,10 +508,16 @@ optionValueOperations(J9PortLibrary *portLibrary, J9VMInitArgs* j9vm_args, IDATA
 				done = TRUE;
 				break;
 
+			case GET_DBL_VALUE:
 			case GET_MEM_VALUE :
 			case GET_INT_VALUE :
 			case GET_PRC_VALUE :
-				*((UDATA*)(reserved)) = 0;				/* Initialize the return value */
+				/* Initialize the return value */
+				if (GET_DBL_VALUE == action) {
+					*(double *)reserved = 0;
```

Comment:
at least should be 0.0
In general:
1)This piece looks ugly.
2)Also (unrelated to this PR): there are many calls of this function with NULL for "reserved".
So this code assumed that for actions listed above "reserved" must be not NULL otherwise it crashed.
Would it be better to add assertion here as well?